sphinxgallery/gen_gallery.py

```diff
@@ -46,12 +48,14 @@ def generate_gallery_rst(app):
     if not plot_gallery:
         return
 
-    build_image_dir = os.path.join(app.builder.outdir, '_images')
-    clean_gallery_out(build_image_dir)
+    #clean_gallery_out(app.builder.outdir)
```

Comment:
This is a new thing. It was commented on purpose but not related to this PR. I was testing If this instruction is at all needed. I was noticing, at least for sphinx gallery documentation, that if you don't clear any image in sphinx 1.2.3. On a new build sphinx wont copy the images again. So maybe this has might not be needed anymore. But It shall be addressed in another PR. For the moment I'll keep deleting images.